.github/workflows/cron-goodbits-sync.yml

```diff
@@ -0,0 +1,36 @@
+name: Cron sync goodbits list
+
+on:
+  schedule:
+    - cron: '13 0,5,10,15,20 * * *'
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths:
+      - 'packages/edge-gateway'
+
+jobs:
+  update:
+    name: Sync goodbits with KV store
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        env: ['staging', 'production']
+    timeout-minutes: 20
```

Comment:
does this job sometimes get stuck? why 20? seems like it should take a few seconds with a list of 1 thing, and may take hours with a list of 1 billion things.

Reply:
no, this was kind of based on my actions template. But I will decrease it for 5 minutes for now